### PR TITLE
Allow specifying start and end date for data

### PR DESCRIPTION
**Pull request by lukipuki**

### CODE REVIEW

File: python/setup_environment.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 set -euo pipefail
 
 rm -rf .venv
-python3.7 -m venv .venv
+python3.7 -m venv --system-site-packages .venv
```
Comment (miskosz) [inline python/setup_environment.sh:5]: Why was this needed?

**Review by miskosz — COMMENTED** [review]
LGTM :)

File: python/setup_environment.sh (hunk 1 above)
Comment (lukipuki) [reply python/setup_environment.sh:5]: This is more for my convenience, I don't want to fill my disk with unnecessary copies of packages :slightly_smiling_face:  Maybe this should have been hidden by an option of the script. I can add a flag `--use-system-packages` or similar if you want.
